server/plugin/api_test.go

```diff
@@ -862,13 +862,27 @@ func Test_handleDateTimeSelection(t *testing.T) {
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
+			mockInterval := int64(1000)
 			p := new(Plugin)
 
 			mockAPI := &plugintest.API{}
 			mockAPI.On("GetBundlePath").Return("mockString", nil)
 			mockAPI.On("LogDebug", testutils.GetMockArgumentsWithType("string", 7)...).Return("LogDebug error")
-			mockAPI.On("LogError", testutils.GetMockArgumentsWithType("string", 6)...).Return("LogError error")
+			mockAPI.On("LogError", mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return("LogError error")
```

Comment:
why this change?

Reply:
Because now we have different types of error